articles/quickstart/spa/_includes/_hosted_login_customization.md

```diff
@@ -1,3 +1,3 @@
 ::: note
-The hosted login page uses the Lock widget and is configured in the [Hosted Pages section](${manage_url}/#/login_page) in your Auth0 dashboard. To customize the look and feel of the Lock widget, see the [configuration options documentation](/libraries/lock/v10/customization).
-:::
+The Auth0 hosted login page uses the Lock widget. You can configure the page in the [Hosted Pages](${manage_url}/#/login_page) section of your Auth0 dashboard. To customize the look and feel of the Lock widget, see the [customization options documentation](/libraries/lock/v10/customization).
```

Comment:
We are missing a link to the main doc https://auth0.com/docs/hosted-pages/login 

And you can omit the 2nd sentence (since the linked doc has all the info on how to configure the HLP)